shelfwind.jl

```diff
@@ -498,10 +493,10 @@ SP_z_yavg = Average(SP_z, dims=(2))
 
 KE_output_fields = (; KE_yavg, ε_yavg, ∫KE, ∫ε, ∫εᴰ, εᴰ_yavg, TKE_yavg, SP_x_yavg, SP_y_yavg, SP_z_yavg  )
 
-simulation.output_writers[:nc] = NetCDFOutputWriter(model, KE_output_fields,
-                                                    filename = "KE_yavg.nc",
-                                                    schedule = TimeInterval(86400second),
-                                                    overwrite_existing = overwrite_existing)
+#simulation.output_writers[:nc] = NetCDFOutputWriter(model, KE_output_fields,
```

Comment:
Something in this output wasn't running for me. You have to try pruning things to see what is causing the error. I think this is the only thing left to fix.